src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/win/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -43,6 +44,16 @@
     <File Include="$(PlaceholderFile)">
       <TargetPath>ref/netstandard1.0</TargetPath>
     </File>
+    <!-- Symbols -->
+    <ArchitectureSpecificNativeSymbol Include="@(ArchitectureSpecificNativeFileAndSymbol -> '%(RelativeDir)PDB\%(FileName).pdb')" />
+    <ArchitectureSpecificNativeSymbol Include="@(ArchitectureSpecificLibFile -> '%(RelativeDir)PDB\%(FileName).pdb')" />
+    <ArchitectureSpecificNativeSymbol Include="@(ArchitectureSpecificToolFile -> '%(RelativeDir)PDB\%(FileName).pdb')" />
+    <AdditionalSymbolPackageExcludes Include="%2A%2A\%2A.dll" />
```

Comment:
I didn’t know but it makes sense especially for the dbi and the dac dlls.  It will simplify the change considerable.